### PR TITLE
Storage and 5.5 push

### PR DESCRIPTION
**Pull request by Dolaned**
Added in storage upload, cleaned up code and adjusted it to laravel 5.5 with package discovery

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+    |
+    | Here you may specify the default file system to be used for uploading the
+    | images of the adverts, please ensure that these are public facing
+    | filesystems.
+    |
+    | Supported: "public", "s3"
+    |
+    */
+
+    'default_fileSystem' => 'public',
```
Comment (adumskis) [inline config/config.php]: Config should keep same text style (snake case).

File: src/Model/Advert.php [hunk 2]
```diff
@@ -7,6 +7,7 @@
 use Illuminate\Support\Facades\Storage;
 use Illuminate\Support\Facades\Validator;
 use Intervention\Image\Facades\Image;
+use Rymanalu\LaravelSimpleUploader\Support\Uploader;
```
Comment (adumskis) [inline src/Model/Advert.php]: Looks like useless `use`

**Comment by adumskis** [comment]
Thank you very much for your work here! That's grate job! Just fix few small things (commented) and I don't see why I shouldn't merge this.

**Comment by Dolaned** [comment]
Yeah sorry for that useless use I was planning on using another library but it was easier just to keep it simple, also I’m having an issue with the routes registering in my application and your viewed_at variable is never updated, I have actually started another pull request to fix the viewed_at issue, do you have time to continue working on this package / help me with my route issue ? Thanks

**Comment by adumskis** [comment]
Yes, of course I can help you. Just not right now. If you need any help, please contact me by email algirdasdum@gmail.com. Once again, thank you!

**Pull request merged**